molo/profiles/forms.py

```diff
@@ -13,6 +15,30 @@
 from phonenumber_field.formfields import PhoneNumberField
 
 
+REGEX_PHONE = r'.*?(\(?\d{3})? ?[\.-]? ?\d{3} ?[\.-]? ?\d{4}.*?'
+REGEX_EMAIL = r'([\w\.-]+@[\w\.-]+)'
```

Comment:
Let's give people the option to override these in settings. Something like:
```python
from django.conf import settings

REGEX_PHONE = settings.REGEX_PHONE if hasattr(settings, 'REGEX_PHONE') else r'.*?(\(?\d{3})? ?[\.-]? ?\d{3} ?[\.-]? ?\d{4}.*?'
```

See https://docs.djangoproject.com/en/1.9/topics/settings/#using-settings-in-python-code